src/components/AppContainer/AppContainer.tsx

```diff
@@ -15,7 +15,7 @@ export default class AppContainer extends React.Component<NestableBaseComponentP
     const { children, className, theme = defaultTheme } = this.props;
```

Comment:
hey @chrismohr, defaulting to defaultTheme in the AppContainer seems correct. but all of our other components also have this `theme = this.props.theme || defaultTheme`. is this correct, or will every component now default to the default YamUI theme instead of what was passed into the app root?

Reply:
@swese44  The reason we do `theme = this.props.theme || defaultTheme` is to reassure typescript that the theme prop is not null. When the component is decorated with @customizable, it will always have a `theme` prop. I see how this is confusing though.  Any ideas for a better solution?

Reply:
cool thanks. i'm going to push up a commit which updates AppContainer to properly set the incoming theme, and updates ExampleWrapper to only set dark theme if provided (and let AppWrapper do the defaulting)